src/fees/compute_fees.py

```diff
@@ -21,17 +21,39 @@
 class Trade:
     """Class for"""
 
-    order_uid: HexBytes
```

Comment:
This code now deviates sufficiently from the code in the circuit breaker to require actual tests.